static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -282,6 +288,9 @@ const makeABTestVariant = (
             const render = (templateFn: ?EpicTemplate) =>
                 copyPromise
                     .then((copy: AcquisitionsEpicTemplateCopy) => {
+                        if (this.options.isAdSlot) {
+                            return this.options.adSlotEpicTemplate();
```

Comment:
Would it be simpler to use the normal `template` function and just have `isAdSlot` set in the test file?

Reply:
yes! 💯